docs/how-to/uc-boot-parameters.rst

```diff
@@ -0,0 +1,81 @@
+How to configure boot parameters on Ubuntu Core 
+================================================
+
+.. note::
+
+    This guide assumes access to features available exclusively to `dedicated Snap Store`_ users.
+
+Ubuntu Core doesn't have the :code:`/etc/default/grub` file mapped for the kernel parameters, neither the `update-grub`_ utility.
+On Ubuntu Core, the kernel parameters can be modified statically or dynamically.
+Static configurations should be added during the image build; refer to :doc:`../how-to/uc-image-creation`.
+Dynamic configurations can be set `using snap options`_ on a running system.
+This guide focuses on the dynamic configuration method, suitable for iteratively tuning the operating system.
+
+.. Refer to :doc:`../how-to/uc-image-creation` if you want to create a real-time Ubuntu Core image.
+
+Open a terminal and access your real-time Ubuntu Core instance.
```

Comment:
I think we also need documentation on creating a real-time Ubuntu core image without static configurations, to allow iterative tuning. There are currently two how-tos:
- This one (`docs/how-to/uc-boot-parameters.rst`); assuming a real-time Ubuntu Core instance is available.
- The image creation guide (`docs/how-to/uc-image-creation.rst`), explaining how to create the image, but only after customizing a gadget to set static config.

Reply:
PR: https://github.com/canonical/real-time-ubuntu-docs/pull/56